token-lending/program/src/state/reserve.rs

```diff
@@ -759,6 +755,7 @@ impl Pack for Reserve {
             config_fees_host_fee_percentage,
             config_deposit_limit,
             config_borrow_limit,
+            config_fee_receiver,
```

Comment:
should change the comment above so that the order of the summands matches new layout

Reply:
Which comment?

Reply:
oh i see